doc/fips.md

```diff
@@ -0,0 +1,132 @@
+# FIPS support
+
+libspdm 3.0.0 starts adding [FIPS 140-3](https://csrc.nist.gov/publications/detail/fips/140/3/final) support for [Cryptographic Algorithm Validation Program (CAVP)](https://csrc.nist.gov/Projects/cryptographic-algorithm-validation-program). The FIPS boundary is libspdm library plus crypto library.
```

Comment:
>The FIPS boundary is libspdm library plus crypto library.

If this is true, then libspdm should implement integrity selftest and automatically run libspdm_fips_run_selftest() at load (dllmain).

Reply:
OK. Then we can remove this.

Currently, I dont think we support DLL.